texastribune/donations-app
# Polishing

#### What's the plan?

To tweak and polish the app, focusing on responsiveness, performance, browser compatibility, and all of the details.
#### What's this PR do?

Adds labels to all videos; enables asset hashing and sets cache control headers for all assets; adds a favicon; adds commas to donation amounts; makes IE8 and IE9 happy; updates fallbacks in case fonts don't load; ensure fallbacks are in place in case someone has js disabled; uses minified js where possible since Middleman doesn't seem to be minifying on build; sends circle memberships to their own form.
#### Why are we doing this? How does it help us?

Makes the site faster and ensures it'll look its best for the most people visiting, with the goal of a better user experience and more donations.
#### How should this be manually tested?

Visit the app locally and click around. 

Bonus points to view it in webpagetest.org and here as well: https://developers.google.com/speed/pagespeed/insights.

You can also check it out in different browsers in Browserstack if you like.
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?

Performance is increased. Yes, set up asset hashing for images/CSS/JS and cache control headers when synced with S3.
#### What are the relevant tickets?

Basecamp project: https://basecamp.com/1866004/projects/8824386
#### Next steps?

Keep improving.
#### Smells?
#### TODOs:
#### Has the relevant documentation/wiki been updated?
#### Technical debt note
